utils_cv/action_recognition/model.py

```diff
@@ -357,7 +303,9 @@ def train_an_epoch(
                         loss = loss / grad_steps
 
                         if mixed_prec and AMP_AVAILABLE:
```

Comment:
Same as above, should only probably check for mixed_prec and not silently ignore if AMP isn't available.

Reply:
Marked as "hooray" but don't see change